### DISCUSSION TITLE
Reforward or show blocked emails during the time an alias was disabled if a user wants to.

### DISCUSSION THREAD

**Discussion by sambartik** [simple-login/app#453] [Feature Requests]
It would be awesome if simplelogin stored blocked emails for few days before discarding it completely. What do you think?

**Comment by nguyenkims** [comment]
That's a good idea!

**Reply by rehanone** [reply]
> But would Simplelogin be able to ensure the encryption and privacy for any emails that are retained?

I don't think there are any parts of SL solution that does storage encryption at app level. You should be able to do that for your storage volume at the OS level in case you need that.

**Reply by rehanone** [reply]
I meant OS level encryption in a self hosted context. That is where the blocked emails will be retained. I am not sure how the SAS offering stores data.

**Comment by port19x** [comment]
Yeah, kinda annoying that I can't forward such a blocked email manually

**Reply by ErTiOntINITum** [reply]
Honestly this is an amazing idea and should be implemented. A two year old feature request that has many people wanting it and getting a bunch of support for it.

**Comment by rehanone** [comment]
It would be amazing to have a "Retention Period" set for each blocked alias, during which sl retains the email and allows for manually forwarding any selected blocked email. This can be a life saver for cases where you generally get marketing emails but at some point someone sends a real email and calls you asking if you have got the email? Case in point, small law firms that you need for taxes each year would send you marketing material all year. But during tax month, you get actual emails from them. I have seen this pattern very often in law firms. So I really wish to see this feature being implemented. There could be a default *retention period* setting of lets say 7 days, with a possibility to override it per alias bases.
